core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2336,6 +2336,18 @@ class SparkContext(config: SparkConf) extends Logging {
    */
   def defaultMinPartitions: Int = math.min(defaultParallelism, 2)
 
+  /**
+   * Total number of CPU cores of all executors registered in the cluster at the moment.
+   * The number reflects current status of the cluster and can change in the future.
+   */
```

Comment:
Likewise, I am less sure why ` @since 2.4.0` looks conventionally missed here. Let's add it here too.

Reply:
Let's at least leave a `@note` that this feature is experimental.

Reply:
> why @since 2.4.0 looks conventionally missed here.

Just because other methods in the files don't have. It seems this file doesn't follow to common coding style. For example, methods don't end by `()`. See, `defaultMinPartitions` instead of `defaultMinPartitions()`.

I will add `@since` and `@note`

Reply:
> Let's at least leave a @note that this feature is experimental.

What does `experimental` mean for user? unstable? can be changed in the future. When I as an user read the note, how should I change my app to take into account it?

Reply:
that means https://github.com/apache/spark/blob/39e2bad6a866d27c3ca594d15e574a1da3ee84cc/common/tags/src/main/java/org/apache/spark/annotation/Experimental.java